Tasks/Common/artifacts-common/module.json

```diff
@@ -0,0 +1,14 @@
+{
+    "messages": {
```

Comment:
Just so you know, these won't work (only the strings in the task's module.json are loaded as resources). It's a problem with packaging-common too.

Reply:
They appear to work perfectly fine as long as you have a line like 
`tl.setResourcePath(path.join(__dirname, 'node_modules/artifacts-common/module.json'));`
in the task, which appears to be callable multiple times for different directories. That is, both strings from the task and common seem to work fine.

Reply:
I guess I hadn't looked at that function in a long time, looks like it was fixed back in 2017...